mypy.ini

```diff
@@ -0,0 +1,23 @@
+[mypy]
+
+[mypy-context.*]
+ignore_missing_imports = True
+
+[mypy-pam_models.*]
+ignore_missing_imports = True
+
+[mypy-pam_interface.*]
+ignore_missing_imports = True
+
+[mypy-pam_mujoco.*]
+ignore_missing_imports = True
+
+[mypy-o80.*]
+ignore_missing_imports = True
+
+[mypy-o80_pam.*]
+ignore_missing_imports = True
+
+[mypy-shared_memory.*]
+ignore_missing_imports = True
```

Comment:
We have the choice between ignoring all missing imports or specifying each import we want to ignore. I have a slight preference for the second one, as we are a bit more controlling what we are doing. But that's debatable.

Reply:
I'm wondering in which situation this warning is actually useful.  Probably in case of third-party modules where a separate stub-package can be installed (but is currently not).  Is there any other use-case for it?

Reply:
I agree, but that can be useful information, that the stubs are missing for one of our dependencies

Reply:
So I'd say we remove `--ignore-missing-imports` from the github check and instead add `--install-types --non-interactive`.  To my understanding this should automatically install official stubs when available.
Then each package needs a mypy.ini (or can also be added to pyproject.toml), where we can decided about actual settings on a per-package base.

Reply:
sounds good !

Reply:
Just a side note (as I found this out right now):  When using the pyproject.toml instead of a separate mypy.ini, the syntax for this is different and allows listing multiple packages in one section (which makes it much more compact):
```
[[tool.mypy.overrides]]
module = [
    "setuptools",
    "rclpy.*",
    "std_msgs.*",
    "std_srvs.*",
    "robot_interfaces",
    "robot_fingers",
    "trifinger_cameras",
    "trifinger_simulation.*",
]
ignore_missing_imports = true
```

Reply:
Thanks ! will update

Reply:
For reference: https://mypy.readthedocs.io/en/stable/config_file.html#example-pyproject-toml

Reply:
I directly pushed to master